app/controllers/defendants_controller.rb

```diff
@@ -13,6 +13,16 @@ def show
                    defendant_path(defendant.arrest_summons_number || defendant.national_insurance_number)
   end
 
+  def remove_link
+    defendant.update(
+      user_name: 'example',
+      unlink_reason_code: 1,
+      unlink_reason_text: 'Wrong MAAT ID'
```

Comment:
regarding using hard-coded values and your question in the PR i would say this if fine for now. 

are their tickets for:
1. retrieving a list of reason codes/descriptions (possibly on the fly from HMCTS via the adaptor)
2. adding fields to the form for user to select  a reason

Reply:
We've got https://dsdmoj.atlassian.net/browse/CBO-1270, though this doesn't cover the usernames part. 

I could add "Send the right usernames with unlink requests" as an extra ticket or add it to that, not sure what is best - I've asked Deepika/Anne-Marie on slack.

Reply:
To clarify: the reason codes + username is not required by HMCTS, but only by our internal systems, as CP does not have the ability to unlink yet.